applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -502,7 +535,16 @@ static void discover_sink_cb(struct bt_conn *conn, struct bt_codec *codec, struc
 		return;
 	}
 
+	/* At this point the location/channel index of the headset is always known */
+	memcpy(headsets[channel_index].sink_codec_cap, temp_codec_cap, sizeof(temp_codec_cap));
+
 	if (ep != NULL) {
+		/* params->num_eps is an increasing index that starts at 0 */
+		if (params->num_eps > 0) {
+			LOG_WRN("The headset has more than one sink endpoint");
```

Comment:
Hmm, the gateway currently supports one sink and one source endpoint per headset.
With this change the gateway will always use the first sink/source endpoint in each headset.
This will work fine, but if the headset has more than one endpoint, the other ones will not be configured/used.

Changed log to:
"The headset has more than one sink/source endpoint, ep index 0 is used by default"